content/en/Platform Deep Dive/Cobalt Account/account-recovery.md

```diff
@@ -161,6 +190,6 @@ If you have access to your Cobalt account, do the following:
 
 Here are the most common error messages you may see upon signing in to Cobalt.
 
-| Error | Description | Troubleshooting Steps |
+| Error | Details | Troubleshooting Steps |
 |---|---|---|
-| `You have to confirm your email address before continuing` | You haven't set up your account on the Cobalt platform. | To fully set up your account: <ol><li>Confirm your email address by selecting the link in the email invitation.</li><li>Create a password.</li></ol><p>Check your spam folder and filters for a welcome email from Cobalt.</p><p>If the invitation link in the email has expired, ask your [Organization Owner](/getting-started/glossary/#organization-owner) to invite you to the organization once again.</p><ul><li>**If you're an Organization Owner**: To resend an invitation to a user, [remove them from your organization](/platform-deep-dive/organization/manage-users/#remove-users), and then [invite](/platform-deep-dive/organization/manage-users/#invite-users) them again.</li><li>**If you're the only Organization Owner in your organization**: To get help, contact {{% csm-support %}}.</li></ul> |
+| `You have to confirm your email address before continuing` | You haven't fully set up your account on the Cobalt platform. | Confirm your email address by clicking the link in the email invitation, and then create a password. Learn more about [setting up your account](#your-account-isnt-fully-set-up). |
```

Comment:
Should we add the invalid email or password error here?

Reply:
Option are to make sure emails correct and is registered. Reset your password using the forgot password flow.

Reply:
Added this error. We'll redirect users to the corresponding section on this page.